libpod/runtime_pod_pause_linux.go

```diff
@@ -0,0 +1,59 @@
+// +build linux
+
+package libpod
+
+import (
+	"context"
+	"path/filepath"
+
+	"github.com/opencontainers/runtime-tools/generate"
+	"github.com/projectatomic/libpod/libpod/image"
+)
+
+func (r *Runtime) getPauseContainer(ctx context.Context, p *Pod, imgName, imgID string) error {
+
+	// Set up generator for pause container defaults
+	g, err := generate.New("linux")
+	if err != nil {
+		return err
+	}
+
+	root := r.config.TmpDir
+	podInfraRootfs := filepath.Join(root, "graph/vfs/pause")
+	g.SetRootPath(filepath.Join(podInfraRootfs, "rootfs"))
```

Comment:
This is unnecessary - newContainer() will take care of this. We can optimize later (pause containers are all the same image and read-only, so we can probably share c/storage for them), but for now we can rely on newContainer to make and set the rootfs